src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -29,3 +29,16 @@
     type: command
     short-summary: Download the DCOS/Kubernetes command line.
 """
+helps['acs create'] = """
+    examples:
+        - name: Create a default acs cluster
+          text: az acs create -g <resource_group_name> -n <acs_cluster_name>
+        - name: Create a Kubernetes cluster
+          text: az acs create --orchestrator-type Kubernetes -g <resource_group_name> -n <acs_cluster_name>
+        - name: Create a Kubernetes cluster with ssh key provided
+          text: az acs create --orchestrator-type Kubernetes -g <resource_group_name> -n <acs_cluster_name> --ssh-key-value <ssh_key_value_or_path>
+        - name: Create a acs cluster with two agent pools
+          text: az acs create -g <resource_group_name> -n <acs_cluster_name> --agent-profiles "[{'name':'agentpool1'},{'name':'agentpool2'}]"
```

Comment:
`<resource_group_name>` -> `MyResourceGroup`
e.g. https://docs.microsoft.com/en-us/cli/azure/acs#create

Same with the other params here.

I think docs prefer that format as it can then be used directly whereas the angle brackets probably wouldn't be accepted by the service.